src/nvim/main.c

```diff
@@ -1396,6 +1396,24 @@ static void load_plugins(void)
 }
 
 /*
+ * Read all the early plugin files (those are sourced before the vimrc)
+ */
+static void load_early_plugins(mparm_T *parmp)
+{
+  if (parmp->use_vimrc != NULL) {
+    if (strcmp(parmp->use_vimrc, "NONE") == 0
+        || strcmp(parmp->use_vimrc, "NORC") == 0) {
+      if (parmp->use_vimrc[2] == 'N')
```

Comment:
Indeed we don't seem to need to. This piece of the code is something I pasted there just to test if this was the issue I was having with `-u NONE`. I should move the check that sets p_lpl around [here](https://github.com/fmoralesc/neovim/blob/early/extra-keymaps/src/nvim/main.c#L297).